.github/workflows/pytest.yml

```diff
@@ -84,6 +84,8 @@ jobs:
           cache-dependency-glob: "pyproject.toml"
       - name: install-reqs
         run: uv pip install -e ".[modin, dask]" --group core-tests --group extra --system
+      - name: install duckdb nightly
+        run: uv pip install -U --pre duckdb --system
```

Comment:
Should we wait for the duckdb release before merging this PR? You are ahead of them 😂

Reply:
nah let's be one step ahead so when they release we're ready 🔥